tests/libwait/wait.go

```diff
@@ -92,10 +92,10 @@ func waitForVMIPhase(ctx context.Context, phases []v1.VirtualMachineInstancePhas
 		vmi, err = virtClient.VirtualMachineInstance(vmi.Namespace).Get(context.Background(), vmi.Name, &metav1.GetOptions{})
 		gomega.ExpectWithOffset(1, err).ToNot(gomega.HaveOccurred())
 
-		gomega.Expect(vmi.Status.Phase == v1.Succeeded).To(gomega.BeFalse(), "VMI %s unexpectedly stopped. State: %s", vmi.Name, vmi.Status.Phase)
+		gomega.Expect(vmi.Status.Phase).ToNot(gomega.Equal(v1.Succeeded), "VMI %s unexpectedly stopped. State: %s", vmi.Name, vmi.Status.Phase)
```

Comment:
Here we can use the custom matcher `BeInPhase`

Reply:
Done